### PR TITLE
switching Jarvis from php 5.6 to 7.0

### PR DESCRIPTION
**Pull request by eric-chau**
This PR aims to switch Jarvis to use PHP 7.0 exclusively.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
     "name": "eric-chau/jarvis",
-    "description": "Jarvis is a PHP micro-framework",
+    "description": "Jarvis is a PHP 7 micro-framework",
     "license": "MIT",
     "require": {
-        "php": ">=5.6.0",
+        "php": ">=7.0.0",
         "nikic/fast-route": "0.*",
-        "symfony/http-foundation": "2.7.*"
+        "symfony/http-foundation": "3.0.*"
```
Comment (mickaelandrieu) [inline composer.json:8]: should be ^3.0

**Comment by mickaelandrieu** [comment]
can you please tag a release ? maybe an `0.1`

**Comment by eric-chau** [comment]
@mickaelandrieu, you want a tag for the php 5.6 version or for the php 7.0 version?

File: composer.json (hunk 1 above)
Comment (eric-chau) [inline composer.json:8]: what does `^3.0` mean?

Comment (mickaelandrieu) [inline composer.json:8]: https://getcomposer.org/doc/articles/versions.md#next-significant-release-operators

(sorry,  I can't explain well by myself :/)

**Comment by mickaelandrieu** [comment]
@eric-chau hmm I'd like to have a PHP5 compat' even if you don't maintain it :) 

I still use it for 2 projects (Jarvis Symfony Edition and a Github bot framework) and I'd like to keep them available with PHP 5.6 for now...

**Comment by eric-chau** [comment]
@mickaelandrieu, from the beginning I planned to create a tag for the PHP 5.6 version. I still had in mind that you started some project with Jarvis ;) so from now you can find a `v0.1.0` tag and a `0.1` branch on this repository ^^
Thank you again for your support and contribution!

**Pull request merged**